doc/sflow/sflow_hld.md

```diff
@@ -645,12 +708,15 @@ An sFlowOrch is introduced in the Orchagent to handle configuration requests. Th
 
 Also, it monitors the SFLOW_SESSIONS_TABLE and PORT state to determine sampling rate / speed changes to derive and set the sampling rate for all the interfaces. Ingress/Egress Sampling is enabled on the interfaces based on direction setting. It uses the SAI samplepacket APIs to set each ports's sampling rate.
 
+Finally, sFlowOrch facilitates the creation/deletion of dropped packet sessions as well as get/set of session specific attributes. sFlowOrch sets the genetlink host interface that is to be used by the SAI driver to deliver dropped packet notifications and reason codes.

```

Comment:
@rck-innovium what is the support required from existing orch agent (sflow_orch) ? is the existing API's enough or need to extend it for new Mirror-in-drop / SAI API spec 2.0?

Reply:
Orchagent needs to be extended to invoke the SAI TAM APIs described in \[[https://github.com/opencomputeproject/SAI/blob/master/doc/TAM/SAI-Proposal-TAM-MOD-Localhost.md#50--api-example](https://github.com/opencomputeproject/SAI/blob/master/doc/TAM/SAI-Proposal-TAM-MOD-Localhost.md#50--api-example) SAI Proposal TAM MOD Localhost - API Example\]